coin_prices_from_coingecko/go.mod

```diff
@@ -1,3 +1,11 @@
 module github.com/blocky/as-demo
 
-go 1.22
+go 1.22.6
+
+toolchain go1.22.8
+
+require (
+	github.com/blocky/basm-go-sdk v0.0.0-20250307220910-8acf608334a6 // indirect
```

Comment:
As mentioned in the PR note, we need to update all these after https://github.com/blocky/basm-go-sdk/pull/5 is merged